src/components/chat/ChatLine.js

```diff
@@ -0,0 +1,32 @@
+import React, { PropTypes } from 'react';
+import { Link } from 'react-router';
+import { routes } from '../../constants/routesPaths';
+import * as constraints from '../../utils/constraints';
+import littleSmilies from '../../assets/little-smilies.svg';
+
+const ChatLine = ({ name, image, comment, topic }) => (
+  <div>
+    <div className="left">
+      <img src={littleSmilies}   className="bitmap"/>
+    </div>
+    <div className="toRight">
+      <img src={littleSmilies}   className="bitmap"/>
+    </div>
+    <span className="middle text-left">
+      <b> {name} </b>
```

Comment:
Remove spaces around `{name}` unless you want them to appear in your page. Check for the same thing in other tags in this PR.